app/src/main/java/de/danoeh/antennapod/menuhandler/FeedItemMenuHandler.java

```diff
@@ -78,12 +78,17 @@ public static boolean onPrepareMenu(Menu menu, FeedItem selectedItem) {
             setItemVisibility(menu, R.id.reset_position, false);
         }
 
-        if(!UserPreferences.isEnableAutodownload() || fileDownloaded) {
-            setItemVisibility(menu, R.id.activate_auto_download, false);
-            setItemVisibility(menu, R.id.deactivate_auto_download, false);
-        } else if (selectedItem.getAutoDownload()) {
-            setItemVisibility(menu, R.id.activate_auto_download, false);
+        if (!selectedItem.getFeed().isLocalFeed()) {
+            if (!UserPreferences.isEnableAutodownload() || fileDownloaded) {
```

Comment:
How about making one single condition out of these two? Then it needs less code duplication. The inner `if` already hides both items, so you just need to add the `!selectedItem.getFeed().isLocalFeed()` part using `||`